packages/core/src/actions.ts

```diff
@@ -432,11 +452,30 @@ export function stop<TContext, TEvent extends EventObject>(
 
   return {
     type: ActionTypes.Stop,
+    subtype: 'spawn',
```

Comment:
some tests are still using `stop` over `invokeStop` when testing the `createMachine` API itself - and because of that, some snapshots see `'spawn'` from here. I could adjust the snapshot-like tests... or refactor them to use `invokeStop` and the problem would be gone. I think it's an acceptable solution - not many people are inspecting the shape of the action objects, this is usually handled for them by the interpreter. 

Thoughts?